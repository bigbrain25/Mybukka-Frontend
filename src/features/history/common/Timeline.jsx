import React from 'react';
import PropTypes from 'prop-types';
// import Column from 'Components/grid/Column';
import './timeline.scss';

const Timeline = ({ time, status, icon, classNames }) => (
  <div className="timeline">
    <div className={`timeline-container timeline-left ${classNames}`}>
      <div className="timeline-content">
        <h5 className="font-size-14 suggested-item-price pt-2">{time}</h5>
        <h5 className="font-size-14 suggested-item-price pb-3">
          {status} {icon || <i>&#9992;...</i>}
        </h5>
      </div>
    </div>
  </div>
);

export default Timeline;

Timeline.defaultProps = {
  icon: '',
  classNames: '',
};

Timeline.propTypes = {
  icon: PropTypes.string,
  time: PropTypes.string.isRequired,
  status: PropTypes.string.isRequired,
  classNames: PropTypes.string,
};
