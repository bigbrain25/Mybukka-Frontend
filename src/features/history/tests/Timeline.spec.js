import React from 'react';
import Timeline from '../common/Timeline';

describe('Timeline component', () => {
  const props = {
    time: '27/3/2019 1:28 am',
    status: 'delivery',
    icon: '',
    classNames: 'text-color',
  };

  const wrapper = mount(<Timeline {...props} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
