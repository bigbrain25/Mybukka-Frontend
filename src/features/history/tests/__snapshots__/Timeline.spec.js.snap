// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timeline component renders properly 1`] = `ReactWrapper {}`;
